addons/ballistics/stringtable.xml

```diff
@@ -2315,5 +2315,10 @@
             <Chinesesimp>枪口初速</Chinesesimp>
             <Chinese>槍口初速</Chinese>
         </Key>
+        <Key ID="STR_ACE_Ballistics_statsNoMagazine">
+            <English>No magazine</English>
```

Comment:
maybe `No magazine selected` would be better?
`No magazine` could imply the weapons doesn't use magazines.

Reply:
No weapon can be used without a magazine, plus I have to keep it short.

Reply:
Ok, still gonna look weird to me though. =)

Reply:
maybe "not loaded" then?

Reply:
"Unloaded"

Reply:
"empty"

Reply:
```
        <Key ID="str_a3_cfgdefaultkeyspresets_empty0">
          <Original>Empty</Original>
          <French>Vide</French>
          <Spanish>Vacío</Spanish>
          <Italian>Vuota</Italian>
          <Polish>Puste</Polish>
          <Russian>Пусто</Russian>
          <German>Leer</German>
          <Czech>Prázdné</Czech>
          <English>Empty</English>
          <Chinese>空</Chinese>
          <Portuguese>Vazio</Portuguese>
          <Korean>비어있음</Korean>
          <Chinesesimp>空</Chinesesimp>
          <Turkish>Boş</Turkish>
          <Japanese>空</Japanese>
        </Key>
```

Reply:
```
      <Key ID="str_a3_cfgmarkers_empty0">
        <Original>empty</Original>
        <French>vide</French>
        <Spanish>vacío</Spanish>
        <Italian>vuoto</Italian>
        <Polish>puste</Polish>
        <Russian>пусто</Russian>
        <German>Leer</German>
        <Czech>prázdné</Czech>
        <English>empty</English>
        <Chinese>空</Chinese>
        <Portuguese>vazio</Portuguese>
        <Korean>비어있음</Korean>
        <Chinesesimp>空</Chinesesimp>
        <Turkish>boş</Turkish>
        <Japanese>空</Japanese>
      </Key>
```

Reply:
my favourite:
```
        <Key ID="str_empty">
          <Original>Empty</Original>
          <French>Vide</French>
          <Spanish>Vacío</Spanish>
          <Italian>Vuoto</Italian>
          <Polish>Pusty</Polish>
          <Russian>Пусто</Russian>
          <German>Leer</German>
          <Czech>Prázdný</Czech>
          <English>Empty</English>
          <Chinese>空</Chinese>
          <Portuguese>Vazio</Portuguese>
          <Korean>비어있음</Korean>
          <Chinesesimp>空</Chinesesimp>
          <Turkish>Boş</Turkish>
          <Japanese>空</Japanese>
        </Key>
```

Reply:
Also like:
```
        <Key ID="str_empty_slot">
          <Original>Empty</Original>
          <French>Vide</French>
          <Spanish>Vacío</Spanish>
          <Italian>Vuoto</Italian>
          <Polish>Pusty</Polish>
          <Russian>Пусто</Russian>
          <German>Leer</German>
          <Czech>Prázdný</Czech>
          <English>Empty</English>
          <Chinese>空</Chinese>
          <Portuguese>Vazio</Portuguese>
          <Korean>없음</Korean>
          <Chinesesimp>空</Chinesesimp>
          <Turkish>Boş</Turkish>
          <Japanese>空</Japanese>
        </Key>
```